idd/Energy+.idd.in

```diff
@@ -71820,10 +71820,26 @@ LoadProfile:Plant,
        \type real
        \units m3/s
        \ip-units gal/min
-  A5 ; \field Flow Rate Fraction Schedule Name
+  A5 , \field Flow Rate Fraction Schedule Name
        \required-field
        \type object-list
        \object-list ScheduleNames
+  A6 , \field Plant Loop Fluid Type
+       \required-field
+       \type choice
+       \key Water
+       \key Steam
+       \default Water
+  N2 , \field Degree of SubCooling
+       \note optional, in case of steam loop
+       \units C
+       \minimum 1.0
+       \default 5.0
+  N3 ; \field Degree of Loop SubCooling
+       \note optional, in case of steam loop
+       \units C
+       \minimum 10.0
+       \default 20.0
```

Comment:
Some idd examples of use of the word optional in note:

    \note optional                 <- I don't like this one since it does not give any other guidance
    \note optional (only required for thermal comfort runs)            <- this is better

Better use of note to guide user (these are more typical of fields that connect to other fields):

    \note Used only when Optical Data Type = Spectral
    \note This field is used only when Outdoor Air Method=DetailedSpecification. <- I like this one

So your notes could say:

    \note This field is used only when Plant Loop Fluid Type=Steam. 

It's really up to you since there are no hard rules for notes but it's better to maintain consistency throughout the idd.

Reply:
Thank you for your insight. I'll update the \note to maintain consistency throughout the idd.